graphics/include/ignition/common/Image.hh

```diff
@@ -125,13 +125,13 @@ namespace ignition
       /// \brief Get the image as a data array
       /// \param[out] _data Pointer to a NULL array of char.
       /// \param[out] _count The resulting data array size
-      public: void Data(unsigned char **_data, unsigned int &_count);
+      public: void Data(unsigned char **_data, unsigned int &_count) const;
```

Comment:
Should we make non-geospatial changes in a separate PR? Not sure if this will cause issues with other libs that use these functions and it will be easier to revert if need be

Reply:
good point; I'll make a separate PR for Image changes